operator/docs/lokistack/sop.md

```diff
@@ -136,3 +136,170 @@ A service(s) has crashed.
 
 - Check the logs of the service that is panicking
 - Examine metrics for signs of failure
+
+## Loki Request Latency
+
+### Impact
+
+A service(s) is affected by slow request responses.
+
+### Summary
+
+A service(s) is slower than expected at processing data.
+
+### Severity
+
+`Critical`
+
+### Access Required
+
+- Console access to the cluster
+- Edit access to the deployed operator and Loki namespace:
+  - OpenShift
+    - `openshift-logging` (LokiStack)
+    - `openshift-operators-redhat` (Loki Operator)
+
+### Steps
+
+- Check the logs of all the services
+- Check to ensure that the Loki components can reach the storage
+
+## Loki Tenant Rate Limit
+
+### Impact
+
+A tenant is being rate limited, resulting in potential loss of data.
```

Comment:
It _might_ be fine as is. The reason I say that is because I'm thinking of what happens if the client is constantly getting battered with 429s because the configuration needs to change. What I'm "worried" about is someone reading that that the data loss is not immediate and delay fixing the issue.